idd/Energy+.idd.in

```diff
@@ -72866,6 +72876,87 @@ DemandManager:Thermostats,
        \note can be selected by entering <Zone Name><space><Global Thermostat Object Name>.
        \type object-list
        \object-list ZoneControlThermostaticNames
+       
+  DemandManager:Ventilation,
+       \memo used for demand limiting Controller:OutdoorAir objects.
+       \extensible:1 Duplicate the last field Controller:OutdoorAir Name
+  A1 , \field Name
+       \required-field
+       \type alpha
+       \reference DemandManagerNames
+  A2 , \field Availability Schedule Name
+       \note Availability schedule name for this DR. Schedule value > 0 means the DR is available.
```

Comment:
@rongpengzhang Instead of "DR" is would say "demand manager".  Also, I notice that the schedule field descriptions for all of the other demand managers say "this system" - probably a copy/paste from another object.  Would be good to change all of those to say demand manager instead of system.